app/assets/javascripts/zpotlight/blocks/mirador_block.js

```diff
@@ -46,15 +76,15 @@ SirTrevor.Blocks.Mirador = (function() {
           '<div class="clearfix">',
             '<legend><%= i18n.t("blocks:mirador:source_location:header") %>:</legend>',
             '<label for="<%= blockID + "_source_location_exhibit_label"  %>" class="radio-inline">',
-              '<input type="radio" name="<%= blockID %>_source_location" id="<%= blockID + "_source_location_exhibit_label" %>" data-behavior="source-location-select" value="exhibit"> <%= i18n.t("blocks:mirador:source_location:exhibit:label") %>',
+              '<input type="radio" name="<%= blockID %>_source_location" id="<%= blockID + "_source_location_exhibit_label" %>" data-behavior="source-location-select" value="exhibit" checked> <%= i18n.t("blocks:mirador:source_location:exhibit:label") %>',
             '</label>',
             '<label for="<%= blockID + "_source_location_iiif_label"  %>" class="radio-inline">',
               '<input type="radio" name="<%= blockID %>_source_location" id="<%= blockID + "_source_location_iiif_label" %>" data-behavior="source-location-select" value="iiif"> <%= i18n.t("blocks:mirador:source_location:iiif:label") %>',
             '</label>',
           '</div>',
           '<div class="form-inline" data-source-location="exhibit">',
             '<div class="form-group">',
-              '<input type="text" placeholder="<%= i18n.t("blocks:mirador:source_location:exhibit:placeholder") %>" data-behavior="source-location-input" />',
+              '<input type="text" class="st-input-string form-control item-input-field" data-twitter-typeahead="true" placeholder="<%= i18n.t("blocks:mirador:source_location:exhibit:placeholder")%>"/>',
```

Comment:
Can we add a `label` to this for screenreader folks? I'll add one to the `IIIF manifest` version of the form in #99.

Reply:
Sure thing. Added in latest commit. FWIW, I'm not sure that we have a label upstream in Spotlight? https://github.com/projectblacklight/spotlight/blob/master/app/assets/javascripts/spotlight/block_mixins/autocompleteable.js#L34

Or even a larger question, if the administrative and curatorial screens (mainly SirTrevor) has been evaluated for accessibility. On a quick try, I couldn't even change the SirTrevor widget type using keyboard only commands. Thanks for raising the issue!